src/action_search.rs

```diff
@@ -0,0 +1,22 @@
+use crate::print_package_table;
+use raur::{Package, SearchBy};
+
+fn is_package_ok(pkg: &Package, filter: &str) -> bool {
+	pkg.name.contains(filter) || pkg.description.iter().any(|descr| descr.contains(filter))
```

Comment:
This should be a case insensitive comparison.

Reply:
awesome! Thanks.
Fixed in latest commit.

Reply:
For sake of a clean history. The last commit can easily be squashed into the first via fixup. `git rebase -i HEAD^^^`

Reply:
Done! All good now?:)

Just in case: I don't mind rebasing, squashing etc personally. But it's not an official requirement for RUA pull-requestion as well, at least for now. I'm fine with just small commits and readable commit messages from others, and I'll do what others ask if it's not too effortful (rebasing and fixup-ing is totally OK).